packages/esm-commons-lib/src/components/banner-tags/patient-status-tag.component.tsx

```diff
@@ -17,22 +32,18 @@ export function PatientStatusBannerTag({ patientUuid }) {
   if (isError) {
     return <div>Error fetching family information</div>;
   }
-
   return (
     <>
-      {/* HIV Status Display */}
       {hivStatus === 'positive' && <Tag type="red">{t('hivPositive', 'HIV Positive')}</Tag>}
       {hivStatus === 'negative' && <Tag type="green">{t('hivNegative', 'HIV Negative')}</Tag>}
 
-      {/* Mother Name Display (if patient is under 10) */}
```

Comment:
Are you sure this widget is only used in Namibia?

Reply:
The mother -child linkage happens in the pmtct.. for this case we are displaying the mother name on the  banner for that specific child and vice versa.... I think its only Namibia that uses that ..